backend/ee/danswer/external_permissions/google_drive/group_sync.py

```diff
@@ -105,9 +104,12 @@ def _fetch_group_members_paginated(
 def gdrive_group_sync(
     db_session: Session,
     cc_pair: ConnectorCredentialPair,
-    docs_with_additional_info: list[DocsWithAdditionalInfo],
-    sync_details: dict[str, Any],
 ) -> None:
```

Comment:
not all sources require sync details, so we check this on a source by source basis